pkg/rtc/participant.go

```diff
@@ -1708,6 +1704,14 @@ func (p *ParticipantImpl) GetPendingTrack(trackID livekit.TrackID) *livekit.Trac
 	return nil
 }
 
+func (p *ParticipantImpl) GetICEConnectionDetails() []*types.ICEConnectionDetails {
```

Comment:
`TransportManager` is an embedded structure. So, I think this interface method can be handled directly handled by `TransportManager` and this can be removed.

Reply:
oh awesome! I'll do that